test/integration/verify/controls/aws_amis.rb

```diff
@@ -0,0 +1,25 @@
+title 'Test AWS AMIs in bulk'
```

Comment:
The singular resource control now tests both plural/singular resources.  Suggest we now remove or refine this test because the underlying calls for these are likely to be heavy:
```
  describe aws_amis(is_public: 'true') do
  describe aws_amis(all_amis: 'true') do
```

Reply:
Im using the plural resource to test the singular resource and removed some tests which were similar to what was already there.